pkg/exporter/exporter.go

```diff
@@ -157,6 +157,10 @@ func (e *exporter) collect(ctx context.Context, m *metrics.Metrics) error {
 		return errors.New("Auth0 log client did not return the expected list of Log type")
 	}
 
+	if len(tenantLogEvents) > 0 {
```

Comment:
Nitpick(non-blocking): could you add a comment, like "updating the exporter start time with latest"

Reply:
Done in 41d017f